src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarShippingInformation.tsx

```diff
@@ -11,71 +11,88 @@ export interface ShippingInformationProps {
-          <RouterLink
-            inline
-            to="https://support.artsy.net/s/article/How-are-taxes-customs-VAT-and-import-fees-handled-on-works-listed-with-secure-checkout"
-            target="_blank"
-            rel="noopener noreferrer"
-            onClick={() => {
-              const payload: ClickedOnLearnMore = {
-                action: ActionType.clickedOnLearnMore,
-                context_module: "Sidebar",
-                subject: "Learn more",
-                type: "Link",
-                flow: "Shipping",
-              }
+        {!!taxInfo && (
+          <Text variant="xs" color="black60">
+            {taxInfo.displayText}{" "}
+            <RouterLink
+              inline
+              to="https://support.artsy.net/s/article/How-are-taxes-customs-VAT-and-import-fees-handled-on-works-listed-with-secure-checkout"
+              target="_blank"
+              rel="noopener noreferrer"
+              onClick={() => {
+                const payload: ClickedOnLearnMore = {
+                  action: ActionType.clickedOnLearnMore,
+                  context_module: "Sidebar",
+                  subject: "Learn more",
+                  type: "Link",
+                  flow: "Shipping",
+                }
 
-              trackEvent(payload)
-            }}
-          >
-            {t`artworkPage.sidebar.shippingAndTaxes.taxInformationLearnMore`}
-          </RouterLink>
-        </Text>
+                trackEvent(payload)
+              }}
+            >
+              {taxInfo.moreInfo.displayText}
+            </RouterLink>
+          </Text>
+        )}
       </>
     )
   }
 
   return (
     <>
-      <Text variant="sm" color="black60">
-        {t`artworkPage.sidebar.shippingAndTaxes.taxInformation`}{" "}
-        <RouterLink
-          inline
-          to="https://support.artsy.net/s/article/How-are-taxes-customs-VAT-and-import-fees-handled-on-works-listed-with-secure-checkout"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          {t`artworkPage.sidebar.shippingAndTaxes.taxInformationLearnMore`}
-        </RouterLink>
-      </Text>
-
-      {shippingOrigin && (
-        <Text variant="sm" color="black60">
+      {!!shippingOrigin && (
+        <Text variant="sm" color="black60" mb={0.5}>
           {t`artworkPage.sidebar.shippingAndTaxes.shipsFrom`} {shippingOrigin}
         </Text>
       )}
 
-      {shippingInfo && (
-        <Text variant="sm" color="black60" data-testid="shipping-info">
+      {!!shippingInfo && (
+        <Text variant="xs" color="black60" data-testid="shipping-info">
           {shippingInfo}
         </Text>
       )}
+
+      {!!priceIncludesTaxDisplay && (
+        <Text variant="xs" color="black60">
+          {priceIncludesTaxDisplay}
+        </Text>
+      )}
+
+      {!!taxInfo && (
+        <Text variant="xs" color="black60">
+          {taxInfo.displayText}{" "}
+          <RouterLink
```

Comment:
Notice that the link clicked does not include tracking event. Only the unlisted link does. Not sure if that is intentional.